mappings/MGI_COVIDSymptom_MP_HPO.tsv

```diff
@@ -0,0 +1,357 @@
+#File contains COVID symptoms collected from literature and medical web pages. Three sets of mappings are provides Symptom to MP, MP to HPO, and symptom to HPO. Use of SSSOM terms: exact - terms have the same meaning except that MP term is not limited to human; broad ontology term would be a parent to the symptom or MP term would be a parent to HPO term; narrow - ontology term has a more restricted meaning than the symptom or ignoring species limit MP term would be a child of the HPO term; close terms are almost exact but there is a subtle distinction; related - terms are similar but would not have a parent/child relation in an ontology. Cut off for broad was based on the term having just subtypes of the symptom, i.e. if it matches the general phenotype but doesn't specify the location then it got a broad mapping, but if the phenotype was for a specific cell type and the closest match was just for the general tissue no mapping was made. Match type is human_curated for all													
```

Comment:
This information can be added alongside the rest of the SSSOM header information as a `mapping_set_description`.